### ISSUE TITLE
Quickstart error: omegaconf ConfigKeyErrorr: str interpolation key 'task_dir' not found

### ISSUE THREAD

**Issue by isekulic** [facebookresearch/Mephisto#404]
Hi all, I'm looking forward to using Mephisto. However, I've been battling with Quickstart guide for a day now.

I have installed everything successfully, but when I try to run simple static task with:
`python static_test_script.py`

I get an error:
`omegaconf.errors.ConfigKeyError: str interpolation key 'task_dir' not found`
Full error stack trace:
[https://pastebin.com/fhtswvqc](https://pastebin.com/fhtswvqc)

I'm new at this stuff, but `task_dir` seems to be defined when I run 
`python static_test_script.py -c job`
Output:
```# @package _global_
mephisto:
  blueprint:
    _blueprint_type: static_task
    onboarding_qualification: ???
    block_qualification: ???
    _group: StaticBlueprint
    units_per_assignment: 2
    extra_source_dir: ${task_dir}/server_files/extra_refs
    data_json: ???
    data_jsonl: ???
    data_csv: ${task_dir}/data.csv
    task_source: ${task_dir}/server_files/demo_task.html
    preview_source: ${task_dir}/server_files/demo_preview.html
    onboarding_source: ???
  provider:
    _provider_type: mock
    requester_name: ???
  architect:
    _architect_type: local
    hostname: localhost
    port: '3000'
    server_type: node
  task:
    task_name: html-static-task-example
    task_title: Test static task
    task_description: This is a simple test of static tasks.
    task_reward: 0.3
    task_tags: static,task,testing
    assignment_duration_in_seconds: 1800
    allowed_concurrent: 0
    maximum_units_per_worker: 0
  log_level: info
task_dir: /Users/ivan.sekulic/proj/Mephisto/examples/simple_static_task
```

Environment:
- macOS Catalina
- zsh
- python==3.7.3
- hydra-core==1.0.6
- omegaconf==2.0.6

**Comment by JackUrb** [comment]
Hi @isekulic - sorry for the trouble here! We unknowingly introduced a bug with the example script in #400 (where the resolve key was removed), and I only discovered this yesterday (and the fix is in #403). I'll break that out and merge it separately.

**Comment by JackUrb** [comment]
This should be resolved on master now that #403 is in. Sorry for the inconvenience, and I hope the rest of your experience with it goes well! Feel free to open an issue anytime.
